common/djangoapps/track/shim.py

```diff
@@ -83,6 +83,13 @@ def remove_shim_context(event):
     'edx.video.seeked': 'seek_video',
     'edx.video.transcript.shown': 'show_transcript',
     'edx.video.transcript.hidden': 'hide_transcript',
+
+    # Needs to add the following, but 'event' is seen as a str with them
+    # Mobile team?
+    # 'edx.video.closed_captions.shown': 'edx.video.closed_captions.shown',
+    # 'edx.video.closed_captions.hidden': 'edx.video.closed_captions.hidden',
+    # 'edx.video.language_menu.shown': 'video_show_cc_menu',
+    # 'edx.video.language_menu.hidden': 'video_hide_cc_menu',
```

Comment:
@clrux I recommend you talk to @jcdyer - he did a bunch of work to enable the workflow that I said would be really hard to do.

If you wait for his patch to land you could likely convert the JS event to the new format and then code up a small shim that converts it to the legacy format.